xbmc/music/windows/GUIWindowMusicNav.cpp

```diff
@@ -549,14 +549,14 @@ void CGUIWindowMusicNav::GetContextButtons(int itemNumber, CContextButtons &butt
         // 4. specific per album
         buttons.Add(CONTEXT_BUTTON_SET_CONTENT, 20195);
       }
-      if (item->HasMusicInfoTag() && item->GetMusicInfoTag()->GetArtist().size() > 0)
+      if (item->HasMusicInfoTag() && item->GetMusicInfoTag()->GetArtistString().size() > 0)
```

Comment:
Will do, was just sticking with the existing code style. Seems superficial really.

But what concerns me more is that I have not been able to test this bit of code, I don't have any matching music videos, do you? Or anyone come to that?